features/kinderen.feature

```diff
@@ -136,9 +136,9 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | voornamen | Karel  |
 
   @proxy
-  Rule: Wanneer de geslachtsnaam van het kind onbekend is, wordt het geleverd met type "KindOnbekend" en indicatieOnbekend met waarde true
+  Rule: Wanneer de geslachtsnaam van het kind onbekend is, wordt het geleverd met type "KindOnbekend".
```

Comment:
in #940 is volgens mij juist de onbekend types niet meer te leveren, omdat dit lastig af te leiden is en omdat de businesswaarde daarvan onduidelijk of niet-bestaand is

Reply:
omdat het lastig is vast te stellen of een kind onbekend is, is ook de nieuwe vorm van deze rule niet terecht. Dus helemaal niks over het kind leveren omdat de geslachtsnaam onbekend is, is niet terecht